vertx-grpc-it/pom.xml

```diff
@@ -123,6 +123,15 @@
           <protocArtifact>com.google.protobuf:protoc:${protoc.version}:exe:${os.detected.classifier}</protocArtifact>
           <pluginId>grpc-java</pluginId>
           <pluginArtifact>io.grpc:protoc-gen-grpc-java:${grpc.version}:exe:${os.detected.classifier}</pluginArtifact>
+          <protocPlugins>
+            <protocPlugin>
+              <id>vertx-grpc-protoc2-plugin</id>
```

Comment:
rather protoc-plugin2, otherwise it makes think that it is protoc version 2

Reply:
Okay, valid point